xla/service/gpu/llvm_gpu_backend/gpu_backend_lib_test.cc

```diff
@@ -33,6 +38,44 @@ TEST(UtilsTest, TestGetSmName) {
   ASSERT_EQ(nvptx::GetSmName(cc_next), "sm_90");
 }
 
+struct VersionPair {
+  int ptx_version;
+  int cuda_version;
```

Comment:
Can you use `ToolVersion` instead of an `int` for the CUDA version?

Reply:
ditto